test/irmin-pack/cli/stat.t/run.t

```diff
@@ -0,0 +1,87 @@
+Simplest possible cram test
```

Comment:
Looks like that's left over from https://dune.readthedocs.io/en/stable/tests.html#file-tests :slightly_smiling_face:

Reply:
Indeed, thanks :)